querylog/logger_writer.go

```diff
@@ -36,7 +36,7 @@ func LogEntryFields(entry *LogEntry) logrus.Fields {
 		"response_reason": entry.ResponseReason,
 		"response_type":   entry.ResponseType,
 		"response_code":   entry.ResponseCode,
-		"question_name":   entry.QuestionName,
+		"question_name":   util.Obfuscate(entry.QuestionName),
 		"question_type":   entry.QuestionType,
 		"answer":          entry.Answer,
```

Comment:
Answer is already obfuscated. That's because it's piped through `util.AnswerToString` which obfuscates everything (including query type, amusingly, even though that's clear from the length).